src/deployment-guides/tap.md

```diff
@@ -328,61 +280,28 @@ export TAP_REGISTRY_SERVER=<registry uri>
 export TAP_REGISTRY_USER=<registry user>
 export TAP_REGISTRY_PASSWORD=<registry password>
 export TAP_CNRS_DOMAIN=<cnrs domain>
+export alv_domain=< app live view app domain>
```

Comment:
Please adjust all instances of `alv_domain` with `TAP_ALV_DOMAIN`. As these are exported environment variables they should be namespaced with `TAP_` prefix and it is customary to define environment variables in all-caps.

Reply:
done